src/templates/datagrid.latte

```diff
@@ -72,10 +72,14 @@
 						<th colspan="{$control->getColumnsCount()}" class="ublaboo-datagrid-th-form-inline">
 							{if $hasGroupActions}
 								{block group_actions}
-									{='ublaboo_datagrid.group_actions'|translate}:
+									<span>
+										{='ublaboo_datagrid.group_actions'|translate}:
+									</span>
```

Comment:
Why is it here - the `span` tag? It is a potential BC break..

Reply:
It's for comfortable ability to hide the text via css. E.g.:

![image](https://user-images.githubusercontent.com/2492578/84879508-97168680-b08b-11ea-93bb-e66e0c089ea8.png)